src/main/kotlin/org/rust/lang/core/resolve2/DefCollector.kt

```diff
@@ -469,20 +470,22 @@ class ProcMacroDefInfo(
     override val path: ModPath,
     val procMacroKind: RsProcMacroKind,
     val procMacroArtifact: CargoWorkspaceData.ProcMacroArtifact?,
+    val props: KnownProcMacroProperties,
 ) : MacroDefInfo()
 
 class MacroCallInfo(
     val containingMod: ModData,
     val macroIndex: MacroIndex,
     val path: Array<String>,
-    val body: String,
+    val body: MacroCallBody,
     val bodyHash: HashCode?,  // null for `include!` macro
     val depth: Int,
     /**
      * `srcOffset` - [CratePersistentId]
      * `dstOffset` - index of [MACRO_DOLLAR_CRATE_IDENTIFIER] in [body]
      */
     val dollarCrateMap: RangeMap = RangeMap.EMPTY,
+    val originalItem: Pair<VisItem, Set<Namespace>>? = null,
```

Comment:
Let's add doc comment to clarify that `originalItem` is not null only in case of proc macro and only if this proc macro is not expanded by the plugin